### ISSUE TITLE
The Text Color of Disable Label in High Contrast Mode is Wrong

### ISSUE THREAD

**Issue by Lijing29** [microsoft/fluentui#12877]
### Environment Information

- **Package version(s)**: (fill this out) latest
- **Browser and OS versions**: (fill this out) Chromium Edge 81.0.416.64 

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/lijinglin29/pen/RwWVJXW

#### Actual behavior:

![image](https://user-images.githubusercontent.com/50400285/80327815-e13f7080-886f-11ea-85d8-0879b7b8e7c5.png)

#### Expected behavior:

The text in disabled text color, which is:
![image](https://user-images.githubusercontent.com/50400285/80327887-1e0b6780-8870-11ea-9bdc-c5016d119b1b.png)

**Comment by aneeshack4** [comment]
I'm not sure if the green is truly expected.

**Comment by joschect** [comment]
@Lijing29 I am not able to reproduce this, can you confirm that it's only happening in the Azure theme? If so, which Azure theme? Dark, light? Both?

**Comment by Lijing29** [comment]
When we set the text to be disabled, it should follow the system design in high contrast mode. In black-on-white mode, the disabled text color is the green. In other words, what the disabled text color is depends on the mode and the system setting.
Below figure is from https://docs.microsoft.com/en-us/windows/uwp/design/accessibility/high-contrast-themes
![image](https://user-images.githubusercontent.com/50400285/80434748-3ab9a500-892d-11ea-8a1b-a5f4156a80e5.png)

Another useful doc is https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-ms-high-contrast

**Comment by Lijing29** [comment]
> @Lijing29 I am able to reproduce this, I'm guessing there is something wrong in the azure theme because this seems to work fine with the default theme. We'll take a look.

Thanks~

**Comment by JustSlone** [comment]
@hyoshis, @Jacqueline-ms, @wsmd, - as codeowners of azure-theme, is this something you all could pickup and fix?

**Comment by Jacqueline-ms** [comment]
@JustSlone absolutely, thanks for bringing to our attention. I will follow the high contrast theme doc provided and report back shortly.

**Comment by JustSlone** [comment]
Fixed by #14534, closing
